src/main/resources/META-INF/spring/momo-context-security.xml

```diff
@@ -22,11 +22,16 @@
     <http pattern="/login/build/**" security="none" />
     <http pattern="/login/resources/**" security="none" />
 
+    <!-- Remove security for print requests, where we cannot include csrf tokens -->
+    <http pattern="/print/**" security="none" />
```

Comment:
Maybe we should talk about this at some point. If we only want to disable CSRF for this interface, but still restrict in some (role based way), something like this could (!! -> didnt test) be possible:

```XML
<http pattern="/print/**" access="hasRole('ROLE_USER')">
  <csrf disabled="true"/>
</http>
```